hal/include/HAL/AnalogOutput.h

```diff
@@ -11,11 +11,15 @@
 
 #include "Handles.h"
 
+typedef HalHandle HalAnalogOutputHandle;
```

Comment:
I would recommend that in a later commit all of these typedefs should be moved to Handles.h.  The reason is cross-usage between different HAL modules.